filter_plugins/oo_filters.py

```diff
@@ -377,6 +377,10 @@ def oo_parse_certificate_names(certificates, data_dir, internal_hostnames):
             else:
                 certificate['names'] = []
 
+            # We will always look for named certificates in data_dir regardless of their original path.
+            certificate['certfile'] = os.path.join(data_dir, os.path.basename(certificate['certfile']))
+            certificate['keyfile'] = os.path.join(data_dir, os.path.basename(certificate['keyfile']))
```

Comment:
@detiber I've placed them in `{{ master_certs_subdir }}/named_certificates/` since we don't know the CN of the certificate yet. Also updated the variable name here since `data_dir` is misleading. If we wanted CN in the path I could restructure this to place the certificates in a temporary location, parse them and then finalize the location.